src/Chapter1.hs

```diff
@@ -556,7 +561,8 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel = flip elem "aeiouAEIOU"
```

Comment:
Absolutely correct! And you use eta-reduction 👍🏼 

One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎

Reply:
I understand that elem will be slower since it will iterate through the string. I'll change it to some guards

Reply:
It's not a concern for this task :slightly_smiling_face: Just an interesting fact. Fortunately, such constructions will be optimized in the future GHC version!